blackbox/docs/configuration.txt

```diff
@@ -117,15 +118,25 @@ Translog
 
   Sets size of transaction log prior to flushing.
 
+.. _translog_sync_interval:
+
 **translog.sync_interval**
   | *Default:*    ``5s``
-  | *Runtime:*   ``no``
+  | *Runtime:*   ``yes``
 
-  Setting ``index.translog.sync_interval`` controls the period after which the
+  Setting ``translog.sync_interval`` controls the period after which the
```

Comment:
afaik it would be great to highlight here that this won't do anything as along as the translog.durability is not set to ASYNC? this is a bit what this PR is about imho